### PR TITLE
[v1.5] Propagate upstream operator changes to downstream for v1.5 release

### PR DESCRIPTION
**Pull request by iam-veeramalla**
**What type of PR is this?**
> /kind enhancement

**What does this PR do / why we need it**:
Propagate upstream operator changes to downstream for v1.5 release

**Have you updated the necessary documentation?**
* [ ] Documentation update is required by this PR.
* [ ] Documentation has been updated.

**How to test changes / Special notes to the reviewer**:

### CODE REVIEW

File: config/crd/bases/argoproj.io_argocds.yaml [hunk 1]
```diff
@@ -2211,72 +2238,1254 @@ spec:
...
-                      be accessed using strict TLS validation
-                    type: boolean
-                  version:
-                    description: Version is the ArgoCD Repo Server container image
-                      tag.
-                    type: string
-                  volumeMounts:
-                    description: VolumeMounts adds volumeMounts to the repo server
-                      container
+                  sidecarContainers:
```
Comment (jopit) [inline config/crd/bases/argoproj.io_argocds.yaml:2241]: This is the argocd CRD from the master branch of the argocd-operator, not the one from the release-0.3 branch.

**Review by jopit — CHANGES_REQUESTED** [review]

File: config/crd/bases/argoproj.io_argocds.yaml (hunk 1 above)
Comment (iam-veeramalla) [reply config/crd/bases/argoproj.io_argocds.yaml]: Ack, made required changes.

**Review by jopit — APPROVED** [review]

**Pull request merged**